---
layout: page
---
# HuBMAP Pipelines

HIVE pipelines serve as a common processing framework for data collected from multiple sites / centers. Our primary criteria for prioritizing pipeline development is whether a pipeline can uniformly process data or whether a datatype is generated by multiple HuBMAP sites (even if using different platforms or techniques).
 
Decisions on pipelines we focus on is between the data providing site and pipeline team lead.  Anyone may propose a pipeline for development and should contact the shared pipeline team to make a determination.  The following additional considerations affect prioritization of pipelines:
1.      Does the data fall under the definition of what would contribute to a 3D biomolecular map?
2.      Does the data and metadata meet quality requirements?
3.      Does the data add to what already exists in the portal and would investigators outside the consortium potentially be interested in it?
4.      Does the pipeline development team have the capacity and methods to develop the pipeline?

### [CODEX](https://docs.google.com/document/d/1NJqJmM6ecstE8g3waWDR4n_uQ7RoKkBxT1Kes1Otcag/edit#heading=h.ca5mry4plx6)

The HuBMAP Consortium CODEX pipeline uses [Cytokit](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3055-3) to process CODEX datasets from raw data to OME-TIFF compliant segmentation results and compiled antigen fluorescence images.

- [Git Hub](https://github.com/hubmapconsortium/codex-pipeline)

### [SPRM â€“ Spatial Process & Relationship Modeling](https://docs.google.com/document/d/1c7UR0Pe1newpVhQY2HEFkfV8O7GAj9Vk4XnuSiSnDeY/edit#)

SPRM is a statistical modeling program used to calculate a range of descriptors from multichannel images. It can be used for any type of multichannel 2D or 3D image (e.g., CODEX, IMS).

- [Git Hub](https://github.com/hubmapconsortium/sprm)

### [Single-cell RNA sequencing](https://docs.google.com/document/d/14Fu32w_AjyOzT82m99DzZz5iUEJa5v98IBPns5vlizo/edit?usp=sharing)

HuBMAP single-cell RNA-seq data sets are processed with a two-stage pipeline, using [Salmon](https://combine-lab.github.io/salmon/) for transcript quantification and [Scanpy](https://icb-scanpy.readthedocs-hosted.com/en/stable/) for secondary analysis. This pipeline is implemented in CWL, calling command-line tools encapsulated in Docker containers.

- [Git Hub](https://github.com/hubmapconsortium/salmon-rnaseq)

### [Single-cell ATAC-seq](https://docs.google.com/document/d/1qNy8DQJ4Xn431huHyTydRJCQSF68Cmu06tZtZnAKW6s/edit)

The HuBMAP Consortium uses a three-stage pipeline for scATAC-seq data sets, composed of [SnapTools](https://github.com/r3fang/SnapTools), [SnapATAC](https://github.com/r3fang/SnapATAC), and [chromVAR](https://bioconductor.org/packages/release/bioc/html/chromVAR.html). This pipeline is written in CWL, calling command-line tools encapsulated in Docker containers.

- [Git Hub](https://github.com/hubmapconsortium/sc-atac-seq-pipeline)


### [Bulk RNA sequencing](https://docs.google.com/document/d/1DIQJBM5icoKNEfafQA6Aaio2sAjui3NfO1fkMAThdOU/edit?usp=sharing)

HuBMAP single-cell RNA-seq data sets are processed using [Salmon](https://combine-lab.github.io/salmon/) for transcript quantification. This pipeline is implemented in CWL, calling command-line tools encapsulated in Docker containers.

- [Git Hub](https://github.com/hubmapconsortium/salmon-rnaseq)


### [Bulk ATAC-seq](https://docs.google.com/document/d/1Jwm5V-A3j1fynriwo9Ec9QWEODovR14zUmaaUNf1Pq0/edit?usp=sharing)

The HuBMAP Consortium uses a two-stage pipeline for scATAC-seq data sets, composed of [SnapTools](https://github.com/r3fang/SnapTools) and [MACS2](https://github.com/macs3-project/MACS) This pipeline is written in CWL, calling command-line tools encapsulated in Docker containers.

- [Git Hub](https://github.com/hubmapconsortium/sc-atac-seq-pipeline)

### [Multiome RNA + ATAC](https://docs.google.com/document/d/110Il6Kat09qqjmJnIEQPcIpoyCJKJYNPlrqy-bkwI7I/edit?usp=sharing)

The HuBMAP pipeline for processing 10x Multiome and snareseq data uses salmon for RNA quantification, HISAT-2 for short read alignment of epigenomic data, Muon for downstream analysis, and Azimuth for cell type annotation.  This pipeline is written in CWL, calling command-line tools encapsulated in Docker containers. 

- [Git Hub](https://github.com/hubmapconsortium/multiome-rna-atac-pipeline/)

### [Visium w/o probes](https://docs.google.com/document/d/1t-TlkecYlp6Nq5wXKgGZrfdNWL7OiHWvlzZ3EXA1Fxs/edit?usp=sharing)

The HuBMAP pipeline for processing whole-transcriptome Visium data uses salmon for quantification, scanpy and squidpy for downstream analysis including spatial analysis.  This pipeline is written in CWL, calling command-line tools encapsulated in Docker containers. 

- [Git Hub](https://github.com/hubmapconsortium/salmon-rnaseq/)

### [Visium w/ probes](https://docs.google.com/document/d/1VCy7fsF4F-2KMnAeQ1fpFOojRSYoWlv0hAYiWzPhe2s/edit?usp=sharing)

The HuBMAP pipeline for processing probe-based Visium data uses BWA for short read alignment, scanpy and squidpy for downstream analysis including spatial analysis.  This pipeline is written in CWL, calling command-line tools encapsulated in Docker containers. 

- [Git Hub](https://github.com/hubmapconsortium/visium-pipeline/)

### [scRNA w/ probes](https://docs.google.com/document/d/1LE8l0LSHLpztD4xmons0zEovBhn730WjMGJye34JS3g/edit?usp=sharing)

The HuBMAP pipeline for processing probe based single-cell RNAseq data uses BWA for short read alignment, scanpy for downstream analysis, and Azimuth for cell type annotation.  This pipeline is written in CWL, calling command-line tools encapsulated in Docker containers. 

- [Git Hub](https://github.com/hubmapconsortium/rna-probes-pipeline/)

### [Xenium/CosMx](https://docs.google.com/document/d/1Jwm5V-A3j1fynriwo9Ec9QWEODovR14zUmaaUNf1Pq0/edit?usp=sharing)

The HuBMAP pipeline for processing Xenium and CosMx data uses spatialdata-io to read the vendor outputs into a standardized SpatialData format, scanpy and squidpy for downstream analysis including spatial analysis.  This pipeline is written in CWL, calling command-line tools encapsulated in Docker containers. 

- [Git Hub](https://github.com/hubmapconsortium/xenium-pipeline/)

### [GeoMx NGS](https://docs.google.com/document/d/1Jwm5V-A3j1fynriwo9Ec9QWEODovR14zUmaaUNf1Pq0/edit?usp=sharing)

The HuBMAP pipeline for processing Xenium and CosMx data uses spatialdata-io to read the vendor outputs into a standardized SpatialData format, scanpy and squidpy for downstream analysis including spatial analysis.  This pipeline is written in CWL, calling command-line tools encapsulated in Docker containers. 

- [Git Hub](https://github.com/hubmapconsortium/xenium-pipeline/)

### [PhenoCycler/CellDive](https://docs.google.com/document/d/1Jwm5V-A3j1fynriwo9Ec9QWEODovR14zUmaaUNf1Pq0/edit?usp=sharing)

The HuBMAP pipeline for processing PhenoCycler and CellDive data uses DeepCell for segmentation, SPRM for quantification, secondary analysis including spatial analysis, and DeepCellTypes, Stellar, and RIBCA for cell type annotation.
- [Git Hub](https://github.com/hubmapconsortium/phenocycler-pipeline/)

### [MIBI](https://docs.google.com/document/d/1Jwm5V-A3j1fynriwo9Ec9QWEODovR14zUmaaUNf1Pq0/edit?usp=sharing)

The HuBMAP pipeline for processing MIBI data uses DeepCell for segmentation, SPRM for quantification, secondary analysis including spatial analysis, and DeepCellTypes, Stellar, and RIBCA for cell type annotation.
- [Git Hub](https://github.com/hubmapconsortium/mibi-pipeline/)