---
layout: default
---

# HuBMAP Documentation

<details>
<summary>HIVE Software Engineering Principles</summary>
<ul>
    <li>Our software development teams use a multi-institutional Agile Scrum approach to create HuBMAP technologies deployed using <a href="https://portal.hubmapconsortium.org/docs/infrastructure">microservices in a hybrid cloud</a>. We run daily distributed stand-ups and two week sprint cycles. This enables continuous new deployments of features and enhancements under permissive open source licenses.</li>
    <li>The HuBMAP Portal principally utilizes the following core technologies, frameworks, and languages: Globus (identity federation, data flow), Python (APIs), Javascript (UI), Neo4j (graph databases), Docker (container per micro service), and Airflow (workflows), among others. Core storage and other high performance services run locally at Pittsburgh Supercomputing Center whereas high availability services run on Amazon Web Services.</li> 
    <li>Software issues, enhancement, and feature requests are tracked using a <a href="https://github.com/hubmapconsortium/portal-ui/issues">GitHub issues board</a> that is populated directly by developers and by user feedback via the help desk.</li>
    <li>HuBMAP technology documentation resides in the <a href="https://portal.hubmapconsortium.org/docs">Portal documentation area</a> as well as within <a href="https://github.com/hubmapconsortium/">HuBMAP GitHub repositories</a>. Other locations include our <a href="https://portal.hubmapconsortium.org/docs/apis">API</a> viewable on <a href="https://smart-api.info/ui/0065e419668f3336a40d1f5ab89c6ba3">SmartAPI</a>. We manage our <a href="https://github.com/hubmapconsortium/portal-docs">documentation using markdown</a>.</li>
    <li>HuBMAP technologies use a <a href="https://portal.hubmapconsortium.org/docs/infrastructure">microservices architecture</a> and is driven by the <a href="https://github.com/hubmapconsortium/gateway#readme">API Gateway</a>, <a href="https://github.com/hubmapconsortium/entity-api#readme">Provenance services</a>, and Pipeline Container Orchestration.</li>
    <li>We maintain dev, test, and production instances of most HuBMAP systems. In some areas we use continuous integration with <a href="https://travis-ci.org/">Travis CI</a> or <a href="https://docs.github.com/en/actions/guides/about-continuous-integration">GitHub CI</a></li>
</ul>
</details>

<details>
<summary>HuBMAP Data Ingest</summary>
<ul>    
    <li>HuBMAP HIVE is responsible for producing and managing data ingest processes and associated software in collaboration with the Data Providers. HuBMAP Data Providers are responsible for producing data and metadata in collaboration with the HIVE. These processes are rapidly evolving into scalable pipelines.</li>
    <li>The core ingest software and UI includes: the <a href="https://github.com/hubmapconsortium/ingest-ui#readme">Data ingest tool</a> (data & metadata, sample, assay, antibody report, contributor upload), <a href="https://github.com/hubmapconsortium/manual-data-ingest">Manual dataset ingest utilities</a>, <a href="https://github.com/hubmapconsortium/airflow#readme">Workflow management</a> + <a href="https://github.com/hubmapconsortium/cwltool">Common Workflow Language</a> tool, individual data pipelines, <a href="https://hubmapconsortium.github.io/ccf/">common coordinate framework / spatial registration via RUI</a>, with Federated identity management and <a href="https://docs.globus.org/">file transfer via Globus</a>.</li>
    <li>HuBMAP metadata is ingested into a Dockerized <a href="https://github.com/hubmapconsortium/neo4j-docker#readme">Neo4j graph database for Provenance</a> as well as various function-specific relational and no-sql databases.</li>
    <li>Data providers submit data using a combination of web registration forms, tools noted above, and registration of experimental and sample protocols at <a href="https://www.protocols.io/workspaces/human-biomolecular-atlas-program-hubmap-method-development">Protocols.io</a>. Metadata is submitted through the ingest process as <a href="https://github.com/hubmapconsortium/ingest-validation-tools">Tab separated value (.TSV) files</a> containing sample, assay, antibody, and contributor metadata that meets HuBMAP specifications.</li>
    <li>The <a href="https://github.com/hubmapconsortium/UUID-api#readme">UUID API</a> forms the basis of ID generation. Data providers use the <a href="https://ingest.hubmapconsortium.org/">Tissue & donor registration tool</a> to generate donor, organ, tissue sample (including spatial data), and dataset-specific identifiers that are interlinked and displayed on the Portal.</li>
    <li>We accept Donor data on a HIPAA conforming Globus site and de-identify Donor data using <a href="http://rio.pitt.edu/services">professional de-identification services</a> via manual abstraction from organ procurement organizations, DICOM data, electronic health record and other tabular data, as available.</li>
    <li>Our antibody validation database and query system (pending release) includes antibody validations done by RRID by assay by organ. For individual datasets data contributors will include the RRID (and related information) for each imaging channel in antibody tab separated values files enabling linkage of submitted antibodies & their validation reports.</li>
    <li>Each HuBMAP collection, ASCT+B table, and reference object receives its own Digital Object Identifier (DOI) using <a href="https://search.datacite.org/works?query=HuBMAP">HuBMAP’s DOI registration service</a>. Each dataset will have its HuBMAP DOI soon. We produce protocol DOIs via protocols.io and standard publication DOIs via <a href="https://scholar.google.com/citations?user=CtGSN80AAAAJ">HuBMAP Publications</a>.</li>
    <li>The <a href="https://hubmapconsortium.github.io/ccf-ui/rui/">CCF RUI (Registration User Interface)</a> is a tool that supports the registration of a three-dimensional (3D) tissue block within a 3D reference organ. The registration data is used in current versions of the Common Coordinate Framework (CCF, see <a href="https://docs.google.com/document/d/11jKl__ltdDO3PBMHgHpZnIcZTNuxGUpX_94l6CtTP2I/edit?usp=sharing">CCF RUI SOP</a>, <a href="https://github.com/hubmapconsortium/ccf-ui">CCF RUI GitHub repository</a>, <a href="https://www.youtube.com/watch?v=142hGer4xvU">RUI Demo</a>) and the CCF Exploration User Interface (EUI) developed within HuBMAP. The RUI currently supports 11 organs, written in TypeScript using libraries such as: Angular 11, Deck.gl, NGXS, Angular Material, and N3.js.</li>
    <li>We will also associate ontologies for <a href="https://hubmapconsortium.github.io/ccf/">reference organs, anatomical structures, cell types, and biomarkers</a> using <a href="https://github.com/hubmapconsortium/ccf-3d-reference-object-library#readme">CCF reference objects</a>, <a href="https://hubmapconsortium.github.io/ccf-asct-reporter/">ASCT+B tables</a>, and <a href="https://azimuth.hubmapconsortium.org/">Azimuth reference objects</a> with the data ingest items.</li>
</ul>
</details>

<details>
<summary>HuBMAP Data Validation</summary>
<ul>
    <li>HuBMAP Data Validation is a continuously improving process that starts with defining QC/QA standards and establishing definitions for donor, sample and assay metadata. Standards, definitions, metadata schema and data directory schema are created by teams under the Data Coordination Working Group. Metadata schemas are available <a href="https://hubmapconsortium.github.io/ingest-validation-tools/">here</a>, along with Excel templates with dropdowns for data entry.</li>
    <li>Data providers format their data and metadata files according to the metadata and data directory schema specifications for each assay type. Required formats for metadata field input are described in the <a href="https://hubmapconsortium.github.io/ingest-validation-tools/">Github page for each assay-specific metadata schema</a>. Data providers also include the required QA/QC assessments of their data as components of the submission.</li>
    <li>Data providers receive registration and validation guidance using <a href="https://docs.google.com/document/d/1KR2TC2y-NIjbBRHTu0giSZATMUfPKxN_/edit">HuBMAP’s data submission guide (currently v1.0)</a> as well as <a href="https://github.com/hubmapconsortium/ingest-validation-tools">Ingest tool documentation</a>.</li>
    <li><a href="https://github.com/hubmapconsortium/ingest-validation-tools#readme">HuBMAP validation tools</a> written in Python ensure data submissions conform to HuBMAP standards which are shared and documented for data providers to use to run many of HuBMAP’s checks on their own prior to submission. Other services include <a href="https://github.com/hubmapconsortium/tableschema-to-template#readme">Metadata submission conversion</a>, <a href="https://github.com/hubmapconsortium/ingest-validation-tests#readme">ingest validation</a> and base checks (checksum, file type, etc.) as well as <a href="https://portal.hubmapconsortium.org/docs/assays">assay-specific checks</a>.</li>  
    <li>HuBMAP staff conduct 178 (and growing) automated and manual QA/QC checks as part of the data submission & publication process. Manual validation steps are being automated as development capacity allows.</li>
    <li>Prior to publication, each dataset is formally approved by the data-providing institution and one or more HIVE members. Data providers must also confirm the quality of spatial and semantic metadata using the <a href="https://portal.hubmapconsortium.org/ccf-eui">CCF EUI</a>.</li>
</ul>
</details>

<details>
<summary>HIVE Data Processing</summary>
<ul>    
    <li>he following HuBMAP <a href="https://portal.hubmapconsortium.org/docs/pipelines">pipelines</a> are run by the HIVE on data from the Data Providers with their assent to gain maximum consistency and usability of final published datasets produced by HuBMAP: <a href="https://github.com/hubmapconsortium/codex-pipeline#readme">CODEX (Cytokit + SPRM)</a>, <a href="https://github.com/hubmapconsortium/example-pipeline">“Example Pipeline”</a>, <a href="https://github.com/hubmapconsortium/ims-mxif-pipeline#readme">Imaging Mass Spectrometry & MxIF</a>, <a href="https://github.com/hubmapconsortium/sc-atac-seq-pipeline#readme">sc/snATAC-seq</a> (SnapTools, SnapATAC, and chromVAR), <a href="https://github.com/hubmapconsortium/salmon-rnaseq/blob/master/README.rst">sc/snRNA-seq</a> (Salmon, Scanpy, scVelo), <a href="https://github.com/hubmapconsortium/sprm#readme">SPRM</a> (Imaging pipeline), Spatial Transcriptomics (Starfish).</li>
    <li>Pipelines are Dockerized by HIVE or data providers and verified by HIVE and integrated with the other portal components, including these general pipeline tools:<a href="https://github.com/hubmapconsortium/ingest-pipeline#readme">Data ingest pipeline</a>, <a href="https://github.com/hubmapconsortium/cross-dataset-common#readme">Mixed datatype pipeline tools</a>, <a href="https://github.com/hubmapconsortium/ome-tiff-pyramid">OME.TIFF Pyramid</a>, Pipeline visualization (CWL), <a href="https://github.com/hubmapconsortium/pipeline-release-mgmt/blob/master/README.rst">Pipeline deployment</a>. These are run by the HIVE in the process of generating datasets for publication.</li>
    <li>The HuBMAP pipelines generate these data types via these tools: <a href="https://github.com/hubmapconsortium/fastq-utils">Sequencing (FASTQ) file tools</a>, <a href="https://github.com/hubmapconsortium/SnapTools/blob/hubmap-develop/README.md">Sequencing (snap) file tools</a>, <a href="https://github.com/hubmapconsortium/portal-containers#readme">Visualization pre-processing</a>, <a href="https://github.com/hubmapconsortium/vitessce-data#readme">Vitessce pre-processing</a>, <a href="https://github.com/hubmapconsortium/ingest-pipeline">Base QA pipeline</a>. QA metrics service (assay specific pipeline QA metric sharing).</li>
    <li>Each of the pipelines produce data and metadata back to the ingest services to enable management of publication status and controlled access of metadata and datasets. Datasets, once approved, are pushed to published and public status, using custom code which changes the status to public of upstream Provenance entities (e.g., samples, donors) and downstream files (e.g., movement of data to Globus public access endpoints if not protected sequence data).</li>
    <li>We currently manually capture dataset submission & publication efforts including active datasets’ status, target month of publication, and future datasets. We comprehensively track donor, sample, dataset, spatial, pipeline, visualization, antibody, security (identifiably sequencing), protocol, documentation, metadata & QA/QC standards compliance, and data contributors.</li>
    <li>Internally, we regularly update data into a spreadsheet and use our Sankey diagram tool to view HuBMAP’s current and planned state of dataset publication (Figure).</li>
</ul>
<img src="https://lh3.googleusercontent.com/qOmRDIj90de1iOHal4-xpdaowe-CF8DjwGIHBihdyTsvI0gVO9gw1kHJpw7TymGV_zKyS9yuzHe1u0aP_4eaHbMY0dR--U1hQBmHKQEJC64LeudJogRcy8xcqdQRRsft9g" alt="Sequencing spatial transcriptomics">
</details>


<details>
<summary>HuBMAP Data Portal</summary>
<ul>    
    <li><a href="https://github.com/hubmapconsortium/portal-ui#readme">The HuBMAP Data Portal UI</a> is principally a Flask app, using React on the front end and primarily Elasticsearch on the back end, wrapped in a Docker container for deployment using Docker Compose. It is deployed at portal.hubmapconsortium.org. Scientists access summary data, visualizations, and data downloads by dataset on the Portal. Globus facilitates file transfer for local use of data.</li>
    <li>The HuBMAP <a href="https://github.com/hubmapconsortium/portal-style-guide#readme">Portal Style Guide</a> is used for the Data Portal and other HuBMAP sites.</li>
    <li>While HuBMAP published datasets are openly accessible, HuBMAP consortium level access is managed via the HuBMAP profile system and uses Globus authentication for credential checking.</li>
    <li><a href="https://github.com/vitessce/vitessce#readme">The Vitessce Viewer</a> is a visual integration tool for exploration of spatial single cell experiments. Its modular design is optimized for scalable, linked visualizations that support the spatial and non-spatial representation of tissue-, cell- and molecule-level data. Vitessce integrates the Viv library to visualize highly multiplexed, high-resolution, high-bit depth image data directly from OME-TIFF files and Bio-Formats-compatible Zarr stores.</li>
    <li>Multiple opportunities to query the data use these mechanisms: General <a href="https://github.com/hubmapconsortium/search-api/blob/test-release/README.md">Search</a> (Elasticsearch), Query tools and Facets (integrated in UI), and Semantic query (not yet available to Portal users) including by Gene, Cell, Spatial, and Multidimensional; while the <a href="https://github.com/hubmapconsortium/ccf-ui#readme">CCF EUI</a> provides a detailed look at different parts of the human body, including the heart, kidney, and spleen and spatial data query.</li>
    <li>HuBMAP’s APIs support registration and loading of data that complies with HuBMAP data standards and ingest formats as well as core functions underpinning the Portal UI itself. <a href="https://github.com/hubmapconsortium/search-api/">Data Search</a> - Search API is a thin wrapper of the Elasticsearch. It handles data indexing and reindexing into the backend Elasticsearch. <a href="https://github.com/hubmapconsortium/uuid-api/blob/test-release/README.md">Identity system</a> - The uuid-api service is a restful web service used to create and query UUIDs used across HuBMAP.</li> 
    <li>The HuBMAP Portal provides access to cutting-edge tools to help analyze the data such as the <a href="https://github.com/hubmapconsortium/ccf-asct-reporter#readme">ASCT+B Reporter</a> - includes a partonomy tree that presents relationships between various anatomical structures and substructures, that is combined with their respective cell types and biomarkers via a bimodal network - and <a href="https://github.com/satijalab/azimuth#readme">Azimuth</a> - is a Shiny app demonstrating a query-reference mapping algorithm for single-cell data - and the Cells API: <a href="https://github.com/hubmapconsortium/cross_modality_query#readme">backend</a>, <a href="https://github.com/hubmapconsortium/hubmap-api-js-client#readme">js client</a>, <a href="https://github.com/hubmapconsortium/hubmap-api-py-client#readme">py client</a> - with other tolls coming such as the <a href="https://github.com/hubmapconsortium/ontology-api#readme">Knowledge Graph</a> and associated <a href="https://github.com/dbmi-pitt/UMLS-Graph#readme">Schema</a> for Ontology ingest & API services and application and biomedical ontologies</li>
    <li>The HIVE monitors HuBMAP portal activity including usage, download, and limited demographic factors using <a href="https://datastudio.google.com/u/0/reporting/03a48766-c00a-4909-8790-03caf3292010/page/FltfB?s=vJoh994ntRw">Monitoring services</a>. Current State <a href="https://docs.google.com/document/d/1fc3r3JGiXmg3If7aYV_K5BQTVa-yY0U3tAggBknkw_Q/edit?usp=sharing">FAIRness Assessment</a>.</li>
</ul>
</details>

<details>
<summary>HuBMAP Governance & Due Diligence</summary>
<ul>
    <li><a href="https://hubmapconsortium.org/policies/">HuBMAP consortium policies</a> are located on the consortium website and cover associate membership, consent, data sharing, data use, material transfer, publication, and NIH-applicable <a href="https://portal.hubmapconsortium.org/docs/consent">Genomic Data Sharing with HuBMAP data</a>.</li>
    <li>We use <a href="https://portal.hubmapconsortium.org/docs/datasets">three categories of permissions</a> for securing access to HuBMAP data: protected, consortium, and public</li>
    <li>Consortium-level access is driven from an integrated  <a href="https://github.com/hubmapconsortium/member-ui#readme">user registration</a> tool that collects and associates credentials among Members’ institutions, Globus file transfer service, GitHub code repositories, Google Drive document storage, and other services presented via the WordPress based HuBMAP consortium website.</li>
    <li>Any identifiable sequencing data is accessible via dbGaP within 6 months of initial publication on the HuBMAP portal in order to ensure secure access to this sensitive data -- for details, see the <a href="https://github.com/hubmapconsortium/dbgap-submission-scripts#readme">Sequencing data dbGaP submission tool</a></li>
    <li>Data providers and the HIVE are responsible for secure loading and storage of identifiable sequencing data -- generally, the data providers manage administrative interaction with dbGaP and the HIVE (IEC) manages technical interaction & data loading of identifiable sequencing datasets.</li>
</ul>
We also automatically collect and display <a href="https://scholar.google.com/citations?user=CtGSN80AAAAJ&hl=en">HuBMAP-generated and referenced publications</a> using Google Scholar.
</details>

<details>
<summary>HuBMAP Human Reference Atlas Documentation</summary>
<ul>
    <li>The latest documentation for the Human Reference Atlas can be accessed at <a href="https://humanatlas.io/standard-operating-procedures">https://humanatlas.io/standard-operating-procedures</a>.</li>
</ul>
</details>
