# ATACseq

## Current Metadata Attributes
| Attribute | Type      | Description              | Allowable Values |
| ----------- | ----------- | -------------------------- | ------------------ |
|Parent sample ID | type | Unique HuBMAP or SenNet identifier of the sample (i.e., block, section or suspension) used to perform this assay. For example, for a RNAseq assay, the parent would be the suspension, whereas, for one of the imaging assays, the parent would be the tissue section. If an assay comes from multiple parent samples then this should be a comma separated list. Example: HBM386.ZGKG.235, HBM672.MKPK.442 or SNT232.UBHJ.322, SNT329.ALSK.102| value |
|Lab ID | type | An internal field labs can use it to add whatever ID(s) they want or need for dataset validation and tracking. This could be a single ID (e.g., "Visium_9OLC_A4_S1") or a delimited list of IDs (e.g., “9OL; 9OLC.A2; Visium_9OLC_A4_S1”). This field will not be accessible to anyone outside of the consortium and no effort will be made to check if IDs provided by one data provider are also used by another.| value |
|Preparation protocol DOI | type | DOI for the protocols.io page that describes the assay or sample procurment and preparation. For example for an imaging assay, the protocol might include staining of a section through the creation of an OME-TIFF file. In this case the protocol would include any image processing steps required to create the OME-TIFF file. Example: https://dx.doi.org/10.17504/protocols.io.eq2lyno9qvx9/v1| value |
|Dataset type | type | The specific type of dataset being produced.| value |
|Analyte class | type | Analytes are the target molecules being measured with the assay.| value |
|Is targeted? | type | Specifies whether or not a specific molecule(s) is/are targeted for detection/measurement by the assay ("Yes" or "No"). The CODEX analyte is protein.| value |
|Acquisition instrument vendor | type | An acquisition instrument is the device that contains the signal detection hardware and signal processing software. Assays generate signals such as light of various intensities or color or signals representing the molecular mass.| value |
|Acquisition instrument model | type | Manufacturers of an acquisition instrument may offer various versions (models) of that instrument with different features or sensitivities. Differences in features or sensitivities may be relevant to processing or interpretation of the data.| value |
|Source storage duration value | type | How long was the source material (parent) stored, prior to this sample being processed.| value |
|Source storage duration unit | type | The time duration unit of measurement| value |
|Time since acquisition instrument calibration value | type | The amount of time since the acqusition instrument was last serviced by the vendor. This provides a metric for assessing drift in data capture.| value |
|Time since acquisition instrument calibration unit | type | The time unit of measurement| value |
|Contributors path | type | The path to the file with the ORCID IDs for all contributors of this dataset (e.g., "./extras/contributors.tsv" or "./contributors.tsv"). This is an internal metadata field that is just used for ingest.| value |
|Data path | type | The top level directory containing the raw and/or processed data. For a single dataset upload this might be "." where as for a data upload containing multiple datasets, this would be the directory name for the respective dataset. For instance, if the data is within a directory called "TEST001-RK" use syntax "./TEST001-RK" for this field. If there are multiple directory levels, use the format "./TEST001-RK/Run1/Pass2" in which "Pass2" is the subdirectory where the single dataset's data is stored. This is an internal metadata field that is just used for ingest.| value |
|Barcode offset | type | Positions in the read at which the cell or capture spot barcodes start. Cell and capture spot barcodes are, for example, 3 x 8 bp sequences that are spaced by constant sequences (the offsets). First barcode at position 0, then 38, then 76. This should be included when constructing sequencing libraries with a non-commercial kit.| value |
|Barcode read | type | Which read file contains the cell or capture spot barcode. This should be included when constructing sequencing libraries with a non-commercial kit. This field is required if the source material is barcoded. This field is used to determine which analysis pipeline to run.| value |
|Barcode size | type | Length of the cell or capture spot barcode in base pairs. Cell and capture spot barcodes are, for example, 3 x 8 bp sequences that are spaced by constant sequences, the offsets. This should be included when constructing sequencing libraries with a non-commercial kit. This field is required if the source material is barcoded. This field is used to determine which analysis pipeline to run.| value |
|UMI offset | type | Position in the read at which the UMI barcode starts. This should be included when constructing sequencing libraries with a non-commercial kit.| value |
|UMI read | type | Which read file contains the UMI barcode. This should be included when constructing sequencing libraries with a non-commercial kit.| value |
|UMI size | type | Length of the umi barcode in base pairs. This should be included when constructing sequencing libraries with a non-commercial kit. This field is required if UMI are present. This field is used to determine which analysis pipeline to run.| value |
|Assay input entity | type | This is the entity from which the analyte is being captured. For example, for bulk sequencing this would be "tissue", while it would be "single cell" for single cell sequencing. This field is used to determine which analysis pipeline to run.| value |
|Number of input cells or nuclei | type | How many cells or nuclei were input to the assay? This is typically not available for preparations working with bulk tissue.| value |
|Library adapter sequence | type | 5’ and/or 3’ read adapter sequences used as part of the library preparation protocol to render the library compatible with the sequencing protocol and instrumentation. This should be provided as comma-separated list of key:value pairs (adapter name:sequence).| value |
|Library average fragment size | type | Average size of sequencing library fragments estimated via gel electrophoresis or bioanalyzer/tapestation. Numeric value in base pairs (bp).| value |
|Library input amount value | type | The amount of cDNA, after amplification, that was used for library construction.| value |
|Library input amount unit | type | unit of library input amount value| value |
|Library output amount value | type | Total amount (eg. nanograms) of library after the clean-up step of final pcr amplification step. Answer the question: What is the Qubit measured concentration (ng/ul) times the elution volume (ul) after the final clean-up step?| value |
|Library output amount unit | type | Units of library final yield.| value |
|Library concentration value | type | The concentration value of the pooled library samples submitted for sequencing.| value |
|Library concentration unit | type | Unit of library concentration value.| value |
|Library layout | type | Whether the library was generated for single-end or paired end sequencing| value |
|Number of PCR cycles for indexing | type | Number of PCR cycles performed in order to add adapters and amplify the library. This does not include the cDNA amplification which is captured in the "number of iterations of cDNA amplification" field.| value |
|Library preparation kit | type | Reagent kit used for library preparation| value |
|Sample indexing kit | type | Indexes are needed for multiplexing sequencing libraries for simultaneous sequencing (pooling) and proper attachment to the Illumina flowcell. Each indexing kit would have a number of compatible sequences ("sample indexing sets") that are used to label some number of samples (the number of sets depend on the kit).| value |
|Sample indexing set | type | The specific sequencing barcode index set used, selected from the sample indexing kit. Example: For 10X this might be "SI-GA-A1", for Nextera "N505 - CTCCTTAC"| value |
|Is technical replicate? | type | Is the sequencing reaction run in replicate, "Yes" or "No". If "Yes", FASTQ files in dataset need to be merged.| value |
|Expected entity capture count | type | Number of cells, nuclei or capture spots expected to be captured by the assay. For Visium this is the total number of spots covered by tissue, within the capture area.| value |
|Sequencing reagent kit | type | Reagent kit used for sequencing| value |
|Sequencing read format | type | Number of sequencing cycles in each round of sequencing (i.e., Read1, i7 index, i5 index, and Read2). This is reported as a comma-delimited list. Example: For 10X snATAC-seq (R1,Index,R2,R3) this might be: 50,8,16,50. For SNARE-seq2 this might be: 75,94,8,75| value |
|Transposition reagent kit | type | If Tn5 came from a kit, provide the kit details. This should be set to "Not applicable" if a custom or in-house kit was used.| value |
|Transposition method | type | Modality of capturing accessible chromatin molecules. For example, this would be the type of kit that was used.| value |
|Sequencing batch ID | type | The ID for the sequencing run. This could, for example, be the chip ID and should allow users the ability to determine which samples were processed together in a sequencing run. It is recommended that data providers prefix the ID with the center name, to prevent values overlapping across centers.| value |
|Capture batch ID | type | A lab-generated ID to identify which cells were captured at the same time. This would, for example, be an ID to denote which datasets were derived from a single 10X Genomics Chromium Controller run. In the case of the 10X Controller this could be the chip ID and would allow users the ability to determine which samples were processed together in a Chromium controller. It is recommended that data providers prefix the ID with the center name, to prevent values overlapping across centers.| value |
|Preparation instrument vendor | type | The manufacturer of the instrument used to prepare (staining/processing) the sample for the assay. If an automatic slide staining method was indicated this field should list the manufacturer of the instrument.| value |
|Preparation instrument model | type | Manufacturers of a staining system instrument may offer various versions (models) of that instrument with different features. Differences in features or sensitivities may be relevant to processing or interpretation of the data.| value |
|Preparation instrument kit | type | The reagent kit used with the preparation instrument.| value |
|Metadata schema ID | type | The string that serves as the definitive identifier for the metadata schema version and is readily interpretable by computers for data validation and processing. Example: 22bc762a-5020-419d-b170-24253ed9e8d9 | value | | value |
