--- 
layout: page 
---
# FACS

<details markdown="1"><summary>Version 2 (current)</summary>

## Version 2 (current)

| Attribute Name | Type | Description | Allowable Values | Required |
|---------------|------|-------------|------------------|----------|
| Lab ID | textfield | An internal field labs can use it to add whatever ID(s) they want or need for dataset validation and tracking. This could be a single ID (e.g., "Visium_9OLC_A4_S1") or a delimited list of IDs (e.g., “9OL; 9OLC.A2; Visium_9OLC_A4_S1”). This field will not be accessible to anyone outside of the consortium and no effort will be made to check if IDs provided by one data provider are also used by another. |  | False |
| Dataset type | Assigned Value | The specific type of dataset being produced. | ```4i```, ```LC-MS```, ```Thick section Multiphoton MxIF```, ```Light Sheet```, ```ATACseq```, ```Resolve```, ```HiFi-Slide```, ```MPLEx```, ```10X Multiome```, ```MALDI```, ```Pixel-seq```, ```Histology```, ```Cell DIVE```, ```FACS```, ```MS Lipidomics```, ```Visium (no probes)```, ```MUSIC```, ```RNAseq```, ```GeoMx (NGS)```, ```GeoMx (nCounter)```, ```RNAseq (with probes)```, ```Singular Genomics G4X```, ```Molecular Cartography```, ```CosMx```, ```MERFISH```, ```2D Imaging Mass Cytometry```, ```Confocal```, ```seqFISH```, ```DART-FISH```, ```MIBI```, ```Olink```, ```Enhanced Stimulated Raman Spectroscopy (SRS)```, ```DESI```, ```Xenium```, ```CyCIF```, ```SNARE-seq2```, ```nanoSPLITS```, ```Stereo-seq```, ```Visium (with probes)```, ```SIMS```, ```Auto-fluorescence```, ```CyTOF```, ```CosMx Proteomics```, ```DBiT-seq```, ```PhenoCycler```, ```CODEX```, ```Second Harmonic Generation (SHG)```, ```Seq-Scope``` | True |
| Analyte class | Assigned Value | Analytes are the target molecules being measured with the assay. | ```Nucleic acid + protein```, ```Lipid + metabolite```, ```Collagen```, ```RNA```, ```Fluorochrome```, ```DNA```, ```Metabolite```, ```DNA + RNA```, ```Saturated lipid```, ```Lipid```, ```Peptide```, ```Protein```, ```Unsaturated lipid```, ```Endogenous fluorophore```, ```Chromatin```, ```Polysaccharide``` | True |
| Acquisition instrument vendor | Assigned Value | An acquisition instrument is the device that contains the signal detection hardware and signal processing software. Assays generate signals such as light of various intensities or color or signals representing the molecular mass. | ```Complete Genomics```, ```Cytek Biosciences```, ```Thermo Fisher Scientific```, ```Sciex```, ```Vizgen```, ```Leica Microsystems```, ```Akoya Biosciences```, ```Keyence```, ```Andor```, ```Standard BioTools (Fluidigm)```, ```Leica Biosystems```, ```Zeiss Microscopy```, ```Ionpath```, ```Motic```, ```In-House```, ```Evident Scientific (Olympus)```, ```GE Healthcare```, ```Element Biosciences```, ```Hamamatsu```, ```Bruker```, ```Illumina```, ```3DHISTECH```, ```Singular Genomics```, ```Huron Digital Pathology```, ```Resolve Biosciences```, ```NanoString```, ```Cytiva```, ```10x Genomics```, ```Microscopes International```, ```BGI Genomics``` | True |
| Acquisition instrument model | Assigned Value | Manufacturers of an acquisition instrument may offer various versions (models) of that instrument with different features or sensitivities. Differences in features or sensitivities may be relevant to processing or interpretation of the data. | ```NovaSeq X```, ```NovaSeq X Plus```, ```Cytek Northern Lights```, ```Lightsheet 7```, ```Resolve Biosciences Molecular Cartography```, ```timsTOF HT```, ```timsTOF Pro 2```, ```timsTOF Pro```, ```timsTOF Ultra```, ```timsTOF Ultra 2```, ```timsTOF SCP```, ```Axio Scan.Z1```, ```MALDI timsTOF Flex Prototype```, ```CosMx Spatial Molecular Imager```, ```Unknown```, ```MERSCOPE Ultra```, ```Juno System```, ```timsTOF FleX```, ```Custom: Multiphoton```, ```CyTOF XT```, ```Helios```, ```EVOS M7000```, ```Aperio AT2```, ```Phenocycler-Fusion 2.0```, ```Axio Observer 5```, ```Axio Observer 7```, ```Axio Observer 3```, ```NanoZoomer-SQ```, ```NanoZoomer S210```, ```NanoZoomer S60```, ```NanoZoomer S360```, ```DM6 B```, ```MoticEasyScan One```, ```In-House```, ```NextSeq 500```, ```BZ-X710```, ```QTRAP 5500```, ```NextSeq 550```, ```HiSeq 2500```, ```HiSeq 4000```, ```NovaSeq 6000```, ```Q Exactive HF```, ```Orbitrap Fusion Lumos Tribrid```, ```Q Exactive```, ```VS200 Slide Scanner```, ```Not applicable```, ```Orbitrap Eclipse Tribrid```, ```MIBIscope```, ```IN Cell Analyzer 2200```, ```timsTOF FleX MALDI-2``` | True |
| Source storage duration value | numeric | How long was the source material stored, prior to this sample being processed? For assays applied to tissue sections, this would be how long the tissue section (e.g., slide) was stored, prior to the assay beginning (e.g., imaging). For assays applied to suspensions such as sequencing, this would be how long the suspension was stored before library construction began. |  | True |
| Source storage duration unit | Assigned Value | The time duration unit of measurement | ```hour```, ```month```, ```day```, ```minute```, ```year``` | True |
| Time since acquisition instrument calibration value | numeric | The amount of time since the acquisition instrument was last serviced or calibrated. This provides a metric for assessing drift in data capture. |  | False |
| Time since acquisition instrument calibration unit | Assigned Value | The time unit of measurement | ```month```, ```day```, ```year``` | False |
| Preparation protocol DOI | link | DOI for the protocols.io page that describes the assay or sample procurement and preparation. For example for an imaging assay, the protocol might begin with staining of a section and finalize with the creation of an OME-TIFF file. In this case the protocol would include any image processing steps required to create the OME-TIFF file. Example: https://dx.doi.org/10.17504/protocols.io.eq2lyno9qvx9/v1. |  | True |
| Is targeted? | Radio | Specifies whether or not a specific molecule(s) is/are targeted for detection/measurement by the assay ("Yes" or "No"). The CODEX analyte is protein. | ```Yes```, ```No``` | True |
| Contributors path | textfield | The path to the file with the ORCID IDs for all contributors of this dataset (e.g., "./extras/contributors.tsv" or "./contributors.tsv"). This is an internal metadata field that is just used for ingest. |  | True |
| Data path | textfield | The top level directory containing the raw and/or processed data. For a single dataset upload this might be "." where as for a data upload containing multiple datasets, this would be the directory name for the respective dataset. For instance, if the data is within a directory called "TEST001-RK" use syntax "./TEST001-RK" for this field. If there are multiple directory levels, use the format "./TEST001-RK/Run1/Pass2" in which "Pass2" is the subdirectory where the single dataset's data is stored. This is an internal metadata field that is just used for ingest. |  | True |
| Parent sample ID | textfield | Unique HuBMAP or SenNet identifier of the sample (i.e., block, section or suspension) used to perform this assay. For example, for a RNAseq assay, the parent would be the suspension, whereas, for one of the imaging assays, the parent would be the tissue section. If an assay comes from multiple parent samples then this should be a comma separated list. Example: HBM386.ZGKG.235, HBM672.MKPK.442 or SNT232.UBHJ.322, SNT329.ALSK.102 |  | True |
| Is erythrocyte lysis performed? | Radio | Process in which red blood cells (RBCs) are broken down in the sample prior to analysis, thereby allowing researchers to focus primarily on white blood cells (WBCs). | ```Yes```, ```No``` | True |
| Metadata schema ID | textfield | The string that serves as the definitive identifier for the metadata schema version and is readily interpretable by computers for data validation and processing. Example: 22bc762a-5020-419d-b170-24253ed9e8d9 |  | False |
| Antibody reagent kit | Assigned Value | The kit containing the set of antibodies pre-conjugated with different heavy metal isotopes used to simultaneously detect and quantify multiple protein markers on individual cells by attaching these metal-labeled antibodies to specific cellular targets, essentially acting as the key component for labeling cells with the various markers needed for analysis on the CyTOF machine. | ```Standard BioTools; Maxpar Nuclear Antigen Staining Kit; PN 201603```, ```Standard BioTools; Maxpar Phosphoprotein Staining Kit; PN 201604```, ```Standard BioTools; Maxpar Cell Surface Staining Kit; PN 201601```, ```Standard BioTools; Maxpar Cytoplasmic/Secreted Antigen Staining Kit; PN 201602```, ```Custom``` | True |
| Viability reagent kit | Assigned Value | The kit used to differentiate between live and dead cells within a sample by selectively staining dead cells with a dye that can be detected by the instrument, allowing researchers to exclude dead cell data from their analysis and ensure accurate results when studying cell populations. | ```Sigma Aldrich; Cisplatin 25mg; PN P4394```, ```Standard BioTools; Cell-ID Cisplatin-198Pt 100 uL; PN 201198```, ```None```, ```Standard BioTools; Cell-ID Intercalator-103Rh 2,000 um; PN 201103B```, ```Standard BioTools; Cell-ID Cisplatin-196Pt 100 uL; PN 201196```, ```Standard BioTools; Cell-ID Cisplatin 100 uL; PN 201064```, ```Standard BioTools; Cell-ID Intercalator-103Rh 500 um; PN 201103A```, ```Standard BioTools; Cell-ID Cisplatin-194Pt 100 uL; PN 201194```, ```Standard BioTools; Cell-ID Cisplatin-195Pt 100 uL; PN 201195```, ```Custom``` | True |
| Is cell activation performed? | Radio | Process by which ligand is binded to its receptors on a cell, which enhances the cell's ability to respond to various stimuli. | ```Yes```, ```No``` | True |
| Activation stimulus | textfield | Specific type of stimulus used to provoke cell activation. Examples would include PMA/ionomycin or CD28in/brefeldin A. This field is required if "is_cells_activation performed" is Yes. |  | False |
| Is FcR blocking applied? | Radio | Process by which a reagent has been added to the staining procedure to block the binding of antibodies to Fc receptors (FcRs) on cells, preventing non-specific binding and ensuring that only the intended target antigen is detected by the antibodies; essentially, it helps to minimize false positive signals by preventing antibodies from attaching to the cell via their Fc region instead of the antigen-specific binding site. | ```Yes```, ```No``` | True |
| Is heparin used? | Radio | Indicates whether heparin was used ("Yes") or not ("No") during staining to prevent non-specific binding of metal-labeled antibodies to eosinophils to reduce background noise. | ```Yes```, ```No``` | True |
| Loaded cell concentration value | numeric | The number of cells present within a given volume of liquid for the experiment immediately prior to the experiment, essentially indicating how densely packed the cells are in a solution. |  | False |
| Loaded cell concentration unit | Assigned Value | Unit of measure for cell concentration, e.g. cells per milliliter (cells/mL). | ```cells/mL``` | False |
| Instrument calibration bead kit | Assigned Value | A set of beads of known mass intensity used to adjust the settings of a flow cytometer to ensure accurate measurements. | ```Standard BioTools; EQ Six Element Calibration Beads 100 mL; PN 201245```, ```Standard BioTools; EQ Four Element Calibration Beads 100 mL; PN 201078```, ```None```, ```Standard BioTools; CyTOF Calibration Beads; PN 201073```, ```Custom``` | True |
| Calibration kit lot number | textfield | Manufacturer's lot number for the calibration bead kit used for the experiment. |  | True |