---
layout: page
---
## HuBMAP Metadata Attributes by Dataset Type

This is a list of available dataset types (data types from multiple supported assays), with links to the valid metadata attributes for each dataset type.  The linked assay pages list all attributes as they have occurred across any versions of the metadata specification for the given dataset type with the most current, valid set of attributes listed first on the page.

| Datsaset Type | Attributes | Description |
|-------|------------|-------------|
| 4i (Iterative Indirect Immunofluorescence Imaging) | [attributes](4i-(Iterative-Indirect-Immunofluorescence-Imaging)) | 4i, or Iterative Indirect Immunofluorescence Imaging, is a high-resolution technique for imaging proteins within cells and tissues. |
| [Autofluorescence](https://docs.hubmapconsortium.org/assays/af) | [attributes](AutoFluorescence)  |   Exploits endogenous fluorescence in a biological tissue to capture an image. The image can then be used to integrate other images from multiple modalities and to align tissues within a 3D experiment. |
| [ATACseq](https://docs.hubmapconsortium.org/assays/atacseq)| [attributes](ATACseq)  |  Identifies accessible DNA regions by probing open chromatin with hyperactive mutant Tn5 Transposase that inserts sequencing adapters into open regions of the genome. |
| [CODEX](https://docs.hubmapconsortium.org/assays/codex) | [attributes](CODEX)  |  Strategy for generating highly multiplexed images of fluorescently-labeled antigens. |
| CosMx-Proteomics | [attributes](CosMx-Proteomics) | CosMx Proteomics is a technology that enables the high-resolution, spatial analysis of proteins within their native tissue environment. It is part of the CosMx Spatial Molecular Imager (SMI) platform, which provides single-cell and subcellular resolution to map protein expression, cell states, and cell-cell interactions in FFPE and fresh frozen tissue samples. |
| CyCIF | [attributes](CyCIF)  |  CyCIF, or Cyclic Immunofluorescence, is a technique used in microscopy to image multiple protein markers within a single sample. It allows for highly multiplexed immunofluorescence imaging, meaning it can detect a large number of different proteins simultaneously. |
| CyTOF | [attributes](CyTOF)  |  A type of mass cytometry that employs antibodies labeled with heavy metal isotopes and uses time-of-flight mass spectrometry to analyze single cells. |
| DESI | [attributes](DESI)  | |
| Enhanced SRS | [attributes](EnhancedSRS)  | |
| FACS | [attributes](FACS)  |  FACS (Fluorescence-Activated Cell Sorting) is a specialized type of flow cytometry used to separate cells based on their unique fluorescent properties. |
| GeoMx | [attributes](GeoMx)  | |
| HiFi | [attributes](HiFi-Slide)  | |
| Histology | [attributes](Histology)  | |
| [IMC](https://docs.hubmapconsortium.org/assays/imc) | [attributes](IMC)  |Combines standard immunohistochemistry with CyTOF mass cytometry to resolve the cellular localization of up to 40 proteins in a tissue sample. |
| [LC-MS](https://docs.hubmapconsortium.org/assays/lcms) | [attributes](LC-MS)  |  Coupling of liquid chromatography (LC) to mass spectrometry (MS) |
| Light Sheet | [attributes](LightSheet)  | |
| [MALDI-IMS](https://docs.hubmapconsortium.org/assays/maldi-ims) | [attributes](MALDI)  |  Matrix-assisted laser desorption/ionization (MALDI) imaging mass spectrometry (IMS) combines the sensitivity and molecular specificity of MS with the spatial fidelity of classical microscopy. |
| MERFISH | [attributes](MERFISH)  |  |
| MIBI | [attributes](MIBI)  |  |
| MPLEx | [attributes](MPLEx)  | MPLeX, or Metabolite, Protein, and Lipid Extraction, is a protocol for extracting and analyzing metabolites, proteins, and lipids from a single sample, and allows for comprehensive multi-omics measurements, integrating data from genomics, transcriptomics, proteomics, metabolomics, and lipidomics. |
| MUSIC | [attributes](MUSIC)  | |
| Olink | [attributes](Olink)  | Olink utilizes Proseek reagents based on PEA, a Proximity Extension Assay technology, to allow 96 oligonucleotide-labeled antibody pairs to bind to their respective protein targets in the sample. A PCR reporter sequence is formed by a proximity-dependent DNA polymerization event and is subsequently detected and quantified using real-time PCR. |
| [RNAseq](https://docs.hubmapconsortium.org/assays/rnaseq) | [attributes](RNAseq)  | While bulk RNAseq elucidates the average gene expression profile in cells comprising a tissue sample, single-cell RNAseq, employs per-cell and per-molecule barcoding to enable single-cell resolution of the gene expression profile.|
| Pixel-seqV2 | [attributes](Pixel-seqV2)  |  Pixel-seqV2 is a spatial transcriptomics method that utilizes polony gels to capture and sequence RNA, proteins or other molecules in tissues with high resolution. These polony gels are arrays of micron-scale DNA clusters, each containing a unique barcode, allowing for the mapping of molecules within their original spatial context in a tissue, thereby allowing researchers to study the spatial organization of cells and their gene expression profiles within tissues. |
| RNAseq With Probes | [attributes](RNAseqWithProbes)  | |
| Second Harmonic Generation | [attributes](SecondHarmonicGeneration)  | |
| Seq-Scope | [attributes](Seq-Scope)  |  Seq-Scope is a spatial barcoding technology that enables high-resolution, unbiased spatial transcriptomics analysis by repurposing the Illumina sequencing platform, thereby allowing researchers to visualize gene expression at a cellular and subcellular level. |
| [SeqFISH](https://docs.hubmapconsortium.org/assays/seqfish) | [attributes](seqFISH)  |  seqFISH technology allows in situ imaging of multiple mRNAs using barcoding and fluorophore-labelled barcode readout-probes.  |
| SIMS | [attributes](SIMS)  | |
| Slide-seq | [attributes](Slide-seq)  | |
| SnareSeq2 | [attributes](SnareSeq2)  | |
| Thick Section Multiphoton MxIF | [attributes](ThickSectionMultiphotonMxIF)  | |
| Visium HD | [attributes](Visium-HD) |  Visium HD is a high-resolution, whole-transcriptome spatial transcriptomics platform that maps gene expression with near-single-cell resolution across an intact tissue section, unlike traditional methods that lose spatial context through tissue dissociation. |
| Visium No Probes | [attributes](VisiumNoProbes)  | |
| Visium With Probes | [attributes](VisiumWithProbes)  | |
| WGS | [attributes](WGS)  | |
