--- 
layout: page 
---
# MPLEx

<details markdown="1"><summary>Version 2.0 (use this one)</summary>

## Version 2.0 (use this one)

| Attribute Name | Type | Description | Allowable Values | Required |
|---------------|------|-------------|------------------|----------|
| lab_id | Textfield | An internal field labs can use it to add whatever ID(s) they want or need for dataset validation and tracking. This could be a single ID (e.g., "Visium_9OLC_A4_S1") or a delimited list of IDs (e.g., “9OL; 9OLC.A2; Visium_9OLC_A4_S1”). This field will not be accessible to anyone outside of the consortium and no effort will be made to check if IDs provided by one data provider are also used by another. |  | False |
| dataset_type | Assigned Value | The specific type of dataset being produced. | ```Visium HD```, ```4i```, ```LC-MS```, ```Thick section Multiphoton MxIF```, ```Light Sheet```, ```ATACseq```, ```Resolve```, ```HiFi-Slide```, ```MPLEx```, ```10X Multiome```, ```MALDI```, ```Histology```, ```Cell DIVE```, ```FACS```, ```MS Lipidomics```, ```Visium (no probes)```, ```MUSIC```, ```RNAseq```, ```GeoMx (NGS)```, ```GeoMx (nCounter)```, ```RNAseq (with probes)```, ```Singular Genomics G4X```, ```Molecular Cartography```, ```CosMx Transcriptomics```, ```MERFISH```, ```Pixel-seqV2```, ```2D Imaging Mass Cytometry```, ```Confocal```, ```seqFISH```, ```DART-FISH```, ```MIBI```, ```Olink```, ```Enhanced Stimulated Raman Spectroscopy (SRS)```, ```DESI```, ```Xenium```, ```CyCIF```, ```SNARE-seq2```, ```nanoSPLITS```, ```Stereo-seq```, ```Visium (with probes)```, ```SIMS```, ```Auto-fluorescence```, ```CyTOF```, ```CosMx Proteomics```, ```DBiT-seq```, ```PhenoCycler```, ```CODEX```, ```Second Harmonic Generation (SHG)```, ```Seq-Scope``` | True |
| analyte_class | Assigned Value | Analytes are the target molecules being measured with the assay. | ```Nucleic acid + protein```, ```Lipid + metabolite```, ```Collagen```, ```RNA```, ```Fluorochrome```, ```DNA```, ```Metabolite```, ```DNA + RNA```, ```Saturated lipid```, ```Lipid```, ```Peptide```, ```Protein```, ```Unsaturated lipid```, ```Endogenous fluorophore```, ```Chromatin```, ```Polysaccharide``` | True |
| acquisition_instrument_vendor | Assigned Value | An acquisition instrument is the device that contains the signal detection hardware and signal processing software. Assays generate signals such as light of various intensities or color or signals representing the molecular mass. | ```Complete Genomics```, ```Cytek Biosciences```, ```Thermo Fisher Scientific```, ```Sciex```, ```Vizgen```, ```Leica Microsystems```, ```Akoya Biosciences```, ```Keyence```, ```Andor```, ```Standard BioTools (Fluidigm)```, ```Leica Biosystems```, ```Zeiss Microscopy```, ```Ionpath```, ```Motic```, ```In-House```, ```Evident Scientific (Olympus)```, ```GE Healthcare```, ```Element Biosciences```, ```Hamamatsu```, ```Bruker```, ```Illumina```, ```3DHISTECH```, ```Singular Genomics```, ```Huron Digital Pathology```, ```Resolve Biosciences```, ```NanoString```, ```Cytiva```, ```10x Genomics```, ```Microscopes International```, ```BGI Genomics``` | True |
| acquisition_instrument_model | Assigned Value | Manufacturers of an acquisition instrument may offer various versions (models) of that instrument with different features or sensitivities. Differences in features or sensitivities may be relevant to processing or interpretation of the data. | ```NovaSeq X```, ```NovaSeq X Plus```, ```Cytek Northern Lights```, ```Lightsheet 7```, ```Resolve Biosciences Molecular Cartography```, ```timsTOF HT```, ```timsTOF Pro 2```, ```timsTOF Pro```, ```timsTOF Ultra```, ```timsTOF Ultra 2```, ```timsTOF SCP```, ```Axio Scan.Z1```, ```MALDI timsTOF Flex Prototype```, ```CosMx Spatial Molecular Imager```, ```Unknown```, ```MERSCOPE Ultra```, ```Juno System```, ```timsTOF FleX```, ```Custom: Multiphoton```, ```CyTOF XT```, ```Helios```, ```EVOS M7000```, ```Aperio AT2```, ```Phenocycler-Fusion 2.0```, ```Axio Observer 5```, ```Axio Observer 7```, ```Axio Observer 3```, ```NanoZoomer-SQ```, ```NanoZoomer S210```, ```NanoZoomer S60```, ```NanoZoomer S360```, ```DM6 B```, ```MoticEasyScan One```, ```In-House```, ```NextSeq 500```, ```BZ-X710```, ```QTRAP 5500```, ```NextSeq 550```, ```HiSeq 2500```, ```HiSeq 4000```, ```NovaSeq 6000```, ```Q Exactive HF```, ```Orbitrap Fusion Lumos Tribrid```, ```Q Exactive```, ```VS200 Slide Scanner```, ```Not applicable```, ```Orbitrap Eclipse Tribrid```, ```MIBIscope```, ```IN Cell Analyzer 2200```, ```timsTOF FleX MALDI-2``` | True |
| source_storage_duration_value | Numeric | How long was the source material (parent) stored, prior to this sample being processed. |  | True |
| source_storage_duration_unit | Assigned Value | The time duration unit of measurement | ```hour```, ```month```, ```day```, ```minute```, ```year``` | True |
| time_since_acquisition_instrument_calibration_value | Numeric | The amount of time since the acquisition instrument was last serviced or calibrated. This provides a metric for assessing drift in data capture. |  | False |
| time_since_acquisition_instrument_calibration_unit | Assigned Value | The time unit of measurement | ```month```, ```day```, ```year``` | False |
| preparation_protocol_doi | Link | DOI for the protocols.io page that describes the assay or sample procurement and preparation. For example for an imaging assay, the protocol might begin with staining of a section and finalize with the creation of an OME-TIFF file. In this case the protocol would include any image processing steps required to create the OME-TIFF file. Example: https://dx.doi.org/10.17504/protocols.io.eq2lyno9qvx9/v1. |  | True |
| is_targeted | Radio | Specifies whether or not a specific molecule(s) is/are targeted for detection/measurement by the assay ("Yes" or "No"). The CODEX analyte is protein. | ```Yes```, ```No``` | True |
| contributors_path | Textfield | The path to the file with the ORCID IDs for all contributors of this dataset (e.g., "./extras/contributors.tsv" or "./contributors.tsv"). This is an internal metadata field that is just used for ingest. |  | True |
| data_path | Textfield | The top level directory containing the raw and/or processed data. For a single dataset upload this might be "." where as for a data upload containing multiple datasets, this would be the directory name for the respective dataset. For instance, if the data is within a directory called "TEST001-RK" use syntax "./TEST001-RK" for this field. If there are multiple directory levels, use the format "./TEST001-RK/Run1/Pass2" in which "Pass2" is the subdirectory where the single dataset's data is stored. This is an internal metadata field that is just used for ingest. |  | True |
| mass_analysis_polarity | Assigned Value | The polarity of the mass analysis (positive or negative ion modes). | ```Negative and positive ion mode```, ```Positive ion mode```, ```Negative ion mode``` | True |
| mass_to_charge_range_low_value | Numeric | The low value of the scanned mass-to-charge range, for MS1. (unitless) |  | False |
| mass_to_charge_range_high_value | Numeric | The high value of the scanned mass-to-charge range, for MS1. (unitless) |  | False |
| mass_resolving_power | Numeric | The mass resolving power m/∆m, where ∆m is defined as the full width at half-maximum (FWHM) for a given peak with a specified mass-to-charge (m/z). (unitless) |  | False |
| mass_to_charge_resolving_power | Numeric | The peak (m/z) used to calculate the resolving power. |  | False |
| ion_mobility | Assigned Value | Specifies which technology was used for ion mobility spectrometry. Technologies for measuring ion mobility: Traveling Wave Ion Mobility Spectrometry (TWIMS), Trapped Ion Mobility Spectrometry (TIMS), High Field Asymmetric waveform ion Mobility Spectrometry (FAIMS), Drift Tube Ion Mobility Spectrometry (DTIMS), Structures for Lossless Ion Manipulations (SLIM), and cyclic Ion Mobility Spectrometry (cIMS). | ```TIMS```, ```SLIM```, ```FAIMS```, ```DTIMS```, ```cIMS```, ```TWIMS``` | False |
| ms_ionization_technique | Assigned Value | The ionization approach (i.e., sample probing method) for performing imaging mass spectrometry. | ```MALDI```, ```SIMS-C60```, ```LDI```, ```HESI```, ```nanoDESI```, ```MALDI-2```, ```DESI```, ```LA```, ```SIMS-H20```, ```ESI``` | True |
| ms_scan_mode | Assigned Value | MS (mass spectrometry) scan mode refers to the number of steps in the separation of fragments. | ```MS2```, ```MS1```, ```MS3``` | True |
| label_name | Textfield | If the samples were labeled (e.g. TMT), provide the name/ID of the label on this sample. Leave blank if not applicable. |  | False |
| lc_instrument_vendor | Assigned Value | The manufacturer of the instrument used for liquid chromatography. | ```Thermo Fisher Scientific```, ```Sciex```, ```In-House```, ```Agilent Technologies```, ```Waters```, ```Bruker```, ```Evosep``` | False |
| lc_instrument_model | Textfield | The model number/name of the instrument used for liquid chromatography. |  | False |
| lc_column_model | Textfield | The model number/name of the liquid chromatography column. If it is a custom self-packed, pulled tip capillary is used enter “Pulled tip capilary”. |  | False |
| lc_resin | Textfield | Details of the resin used for liquid chromatography, including vendor, particle size, pore size. |  | False |
| lc_column_length_value | Numeric | Liquid chromatography column length. |  | False |
| lc_column_length_unit | Assigned Value | Units for liquid chromatography column length (typically cm). | ```um```, ```mm```, ```cm``` | False |
| lc_temperature_value | Numeric | Liquid chromatography temperature. |  | False |
| lc_inner_diameter_value | Numeric | Liquid chromatography column inner diameter. |  | False |
| lc_flow_rate_value | Numeric | Value of flow rate. |  | False |
| lc_gradient_value | Numeric | Liquid chromatography gradient. |  | False |
| lc_gradient_unit | Assigned Value | Unit for liquid chromatography gradient | ```minute``` | False |
| lc_mobile_phase_a | Textfield | Composition of mobile phase A. |  | False |
| lc_mobile_phase_b | Textfield |  |  | False |
| spatial_sampling_technique | Assigned Value |  | ```nanoSPLITS```, ```nanoPOTS```, ```LESA```, ```microPOTS```, ```LCM```, ```microLESA``` | False |
| spatial_sampling_target | Textfield | Specifies the cell-type or functional tissue unit (FTU) that is targeted in the spatial profiling experiment. Leave blank if data are generated in imaging mode without a specific target structure. |  | False |
| analysis_protocol_doi | Link | A DOI to a protocols.io protocol describing the software and database(s) used to process the raw data. Example: https://dx.doi.org/10.17504/protocols.io.bsu5ney6 |  | True |
| metadata_schema_id | Textfield | The string that serves as the definitive identifier for the metadata schema version and is readily interpretable by computers for data validation and processing. Example: 22bc762a-5020-419d-b170-24253ed9e8d9 |  | True |
| data_collection_mode | Assigned Value | Mode of data collection in tandem MS assays. Either DDA (Data-dependent acquisition), DIA (Data-independent acquisition), SRM (multiple reaction monitoring), or PRM (parallel reaction monitoring). | ```DDA```, ```PRM```, ```DIA```, ```SRM``` | False |
| lc_column_vendor | Assigned Value | The manufacturer of the liquid chromatography column unless self-packed, pulled tip capillary is used. | ```Thermo Fisher Scientific```, ```In-House```, ```Waters```, ```Bruker```, ```Evosep```, ```IonOpticks``` | False |
| lc_temperature_unit | Assigned Value |  | ```celsius``` | False |
| lc_inner_diameter_unit | Assigned Value |  | ```um```, ```mm```, ```cm``` | False |
| lc_flow_rate_unit | Assigned Value | Units of flow rate. | ```nL/min```, ```mL/min``` | False |
| spatial_sampling_type | Assigned Value | Specifies whether or not the analysis was performed in a spatially targeted manner. Spatial profiling experiments target specific tissue foci but do not necessarily generate images. Spatial imaging expriments collect data from a regular array (pixels) that can be visualized as heat maps of ion intensity at each location (molecular images). Leave blank if data are derived from bulk analysis. | ```Imaging```, ```Profiling``` | False |
| parent_sample_id | Textfield | Unique HuBMAP or SenNet identifier of the sample (i.e., block, section or suspension) used to perform this assay. For example, for a RNAseq assay, the parent would be the suspension, whereas, for one of the imaging assays, the parent would be the tissue section. If an assay comes from multiple parent samples then this should be a comma separated list. Example: HBM386.ZGKG.235, HBM672.MKPK.442 or SNT232.UBHJ.322, SNT329.ALSK.102 |  | True |

</details>