---
layout: page
---
## HuBMAP Metadata by Dataset Type

A list of available dataset types (data types from multiple supported assays), with a link (**&alpha;**) to the valid metadata attributes for each dataset type. The linked assay metadata pages list all attributes, as they have occurred, across any versions of the metadata specification for the given dataset type with the most current, valid set of attributes listed first on the page. Directory schemas for each dataset type are also linked via the (**&delta;&sigma;**) icon in the description column.


| Dataset Type | Description |
|--------------|-------------|
| [Autofluorescence (AF)](https://docs.hubmapconsortium.org/assays/af) [&#x2637;](AutoFluorescence "Attribute description") | Exploits endogenous fluorescence in biological tissue to capture an image. The image can be used to integrate other images from multiple modalities and align tissues within a 3D experiment. Link to [AF &#x0064;&#x0073;](https://hubmapconsortium.github.io/ingest-validation-tools/af/current/ "directory schema"). |
| [ATACseq](https://docs.hubmapconsortium.org/assays/atacseq) [&#x269F;](ATACseq "Attribute description") | Assay for Transposase-Accessible Chromatin using sequencing (ATACseq) identifies accessible DNA regions, probing open chromatin with hyperactive mutant Tn5 Transposase that inserts sequencing adapters into open regions of the genome. Link to [ATACsec directory schema](https://hubmapconsortium.github.io/ingest-validation-tools/atacseq/current/). | 
| [CODEX](https://docs.hubmapconsortium.org/assays/codex) [&#x2348;](CODEX "Attribute description") | Co-detection by indexing (CODEX) is a strategy for generating highly multiplexed images of fluorescently-labeled antigens. Link to [CODEX &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/codex/current/ "directory schema"). |
| [DESI](https://pmc.ncbi.nlm.nih.gov/articles/PMC6053038/) [&#x2350;](DESI "Attribute description") | Desorption Electrospray Ionization (DESI), an ambient ionization technique that can be coupled to mass spectrometry (MS) for chemical analysis of samples at atmospheric conditions. Link to [DESI &delta;&sigmaf;](https://hubmapconsortium.github.io/ingest-validation-tools/desi/current/ "directory schema"). |
| [Enhanced SRS](https://www.nature.com/articles/s41467-019-13230-1) [&#x2398;](EnhancedSRS "Attribute description") | Refers to improvements made to Stimulated Raman Scattering (SRS), a technique used in microscopy and spectroscopy for chemical imaging and analysis. These enhancements aim to improve sensitivity, spatial resolution, and other capabilities of SRS. Link to [Enhanced SRS &Delta;&Sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/enhanced-srs/current/ "directory schema"). |
| [GeoMx](https://nanostring.com/products/geomx-digital-spatial-profiler/spatial-multiomics-enabled-with-geomx-dsp/) [&alpha;](GeoMx "Attribute description") | A platform for spatial biology that analyzes RNA and protein expression within tissue sections which allows for non-destructive, in situ profiling of gene expression and protein levels from specific regions of interest (ROIs) within a tissue. Link to [GeoMx &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/geomx-ngs/current/ "directory schema"). |
| [HiFi-Slide](https://www.researchgate.net/publication/370676672_HiFi-Slide_spatial_RNA-Sequencing_v2) [&alpha;](HiFi "Attribute description") | High-Fidelity Spatial Transcriptomic Slide (HiFi-Slide) sequencing, a super-resolution spatial transcriptomics sequencing technology, captures and spatially resolves genome-wide RNA expression in a submicron resolution for fresh-frozen tissue. Link to [HiFi &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/hifi-slide/current/ "directory schema"). | 
| [Histology](https://en.wikipedia.org/wiki/Histology) [&alpha;](Histology "Attribute description") | The microscopic study of tissue composition and structure, often referred to as microscopic anatomy. It involves examining tissue samples, typically after they've been sectioned, stained, and placed under a microscope. Link to [Histology &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/histology/current/ "directory schema"). |
| [IMC](https://docs.hubmapconsortium.org/assays/imc) [&alpha;](IMC "Attribute description") | Combines standard immunohistochemistry with CyTOF mass cytometry to resolve the cellular localization of up to 40 proteins in a tissue sample. Link to [IMC &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/imc-2d/current/ "directory schema"). |
| [LC-MS](https://docs.hubmapconsortium.org/assays/lcms) [&alpha;](LC-MS "Attribute description")| Coupling of liquid chromatography (LC) to mass spectrometry (MS). Link to [LC-MS &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/lcms/current/ "directory schema"). |
| [Light Sheet](https://en.wikipedia.org/wiki/Light_sheet_fluorescence_microscopy) [&alpha;](LightSheet "Attribute description") | A fluorescence imaging technique that uses a thin sheet of laser light to illuminate a sample, allowing for high-resolution, 3D imaging with reduced photobleaching and phototoxicity; particularly useful for imaging large, thick, or delicate biological samples, like developing embryos or organoids. Link to [Light Sheet &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/lightsheet/current/ "directory schema"). |
| [MALDI-IMS](https://docs.hubmapconsortium.org/assays/maldi-ims) [&alpha;](MALDI "Attribute description") | Matrix-assisted laser desorption/ionization (MALDI) imaging mass spectrometry (IMS) combines the sensitivity and molecular specificity of MS with the spatial fidelity of classical microscopy. Link to [MALDI-IMS &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/maldi/current/ "directory schema"). |
| Multiplex Ion Beam Imaging [(MIBI)](https://www.researchgate.net/figure/Multiplexed-ion-beam-imaging-workflow-for-high-resolution-spatial-proteomics-Here_fig1_349770840) [&alpha;](MIBI "Attribute description") | Preserved tissue sections, mounted on conductive substrates are incubated with unique isotopic transition metal-tagged antibody reporters. An oxygen primary ion beam rasters the sample surface, ejecting and ionizing the isotope reporters. Their masses are subsequently measured via a mass analyzer. Link to [MIBI &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/mibi/current/ "directory schema"). |
| Multiplexed Error-Robust Fluorescence In Situ Hybridization [(MERFISH)](https://pubmed.ncbi.nlm.nih.gov/27241748/) [&alpha;](MERFISH "Attribute description") | A spatial transcriptomics technology that allows for the simultaneous imaging of hundreds to thousands of RNA species within single cells, providing both copy number and spatial distribution information. Link to [MERFISH &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/merfish/current/ "directory schema"). |
|Multiplexed Universal Single-cell Chromatin and RNA interactions [(MUSIC)](https://www.nature.com/articles/s41586-024-07239-w) [&alpha;](MUSIC "Attribute description") | A sequencing assay that allows profiling of gene expression, co-complexed DNA sequences, and RNA-chromatin interactions from the same single-cell nucleus. Both RNA and fragmented DNA within a nucleus are labelled with a unique cell barcode, enabling identification and matching of RNA and DNA sequences. Link to [MUSIC &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/music/current/ "directory schema"). |
| [RNAseq](https://docs.hubmapconsortium.org/assays/rnaseq) [&alpha;](RNAseq "Attribute description") | While bulk RNAseq elucidates the average gene expression profile in cells comprising a tissue sample, single-cell RNAseq employs per-cell and per-molecule barcoding to enable single-cell resolution of the gene expression profile. Link to [RNAseq &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/rnaseq/current/ "directory schema").|
| [RNAseq with Probes](https://pmc.ncbi.nlm.nih.gov/articles/PMC5717752/#) [&alpha;](RNAseqWithProbes "Attribute description") | Uses probes to capture and enrich specific regions of the RNA for targeted sequencing, allowing for in-depth analysis of those regions. Link to [RNAseq with Probes &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/rnaseq-with-probes/current/ "directory schema").|
| Second Harmonic Generation [(SHG)](https://en.wikipedia.org/wiki/Second-harmonic_imaging_microscopy) [&alpha;](SecondHarmonicGeneration "Attribute description") | Single-cycle Fluorescence Microscopy (SFM). A technique that utilizes the nonlinear optical phenomenon of SHG to image biological tissues and structures, particularly those containing collagen. Link to [SHG &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/second-harmonic-generation/current/ "directory schema").| 
| Sequenced Fluorescence In Situ Hybridization [(SeqFISH)](https://docs.hubmapconsortium.org/assays/seqfish) [&alpha;](seqFISH "Attribute description") | SeqFISH technology allows in situ imaging of multiple mRNAs using barcoding and fluorophore-labelled barcode readout-probes. Link to [SeqFISH &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/seqfish/ "directory schema"). _The consortium is no longer accepting data of this type_.|
| [SIMS](https://www.frontiersin.org/journals/chemistry/articles/10.3389/fchem.2023.1237408/full) [&alpha;](SIMS "Attribute description") | Secondary-ion mass spectrometry (SIMS) is a technique used to analyze the composition of solid surfaces and thin films by sputtering the surface of the specimen. Link to [SIMS &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/sims/current/ "directory schema").| 
| [Slide-seq](https://www.nature.com/articles/s41587-020-0739-1) [&alpha;](Slide-seq "Attribute description") | Provides a scalable method for obtaining spatially resolved gene expression data at resolutions comparable to the sizes of individual cells. Link to [Slide-seq &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/slide-seq/current/ "directory schema").|
| [SnareSeq2](https://www.nature.com/articles/s41596-021-00507-3) [&alpha;](SnareSeq2 "Attribute description") | This method uses tagmentation within permeabilized and fixed single-nucleus isolates to capture accessible chromatin (AC) regions, followed by the capture and reverse transcription of RNA transcripts. Link to [SnareSeq2 &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/snareseq2/current/ "directory schema").|
| Thick Section Multiphoton [(TSM) MxIF](https://pmc.ncbi.nlm.nih.gov/articles/PMC9959383/#) [&alpha;](ThickSectionMultiphotonMxIF "Attribute description") | One version of MXIF (multiplexed fluorescence microscopy), an imaging platform whereby a large number of cellular and histological markers can be investigated on a single tissue section. Link to [TSM MxIF &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/thick-section-multiphoton-mxif/current/ "directory schema").|
| [Visium No Probes (NP)](https://ostr.ccr.cancer.gov/emerging-technologies/spatial-biology/visium/) [&alpha;](VisiumNoProbes "Attribute description") | A spatial transcriptomics solution that allows researchers to analyze gene expression patterns within the spatial context of a tissue. An in situ method that captures RNA transcripts within the tissue and then sequences them. Link to [Visium NP &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/visium-no-probes/current/ "directory schema"). |
| [Visium with Probes](https://ngisweden.scilifelab.se/methods/10x-genomics-visium-cytassist-for-ffpe-samples/) [&alpha;](VisiumWithProbes "Attribute description") | Offers spatially resolved transcriptomics through the 10X Genomics Visium CytAssist, which combines histology with probe-based transcriptomics in a spatial context. Link to [Visium with Probes &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/visium-with-probes/current/ "directory schema"). |
|Whole Genome Sequencing [(WGS)](https://en.wikipedia.org/wiki/Whole_genome_sequencing) [&alpha;](WGS "Attribute description") | The process of determining the entire DNA sequence of an organism's genome at a single time. This entails sequencing all of an organism's chromosomal and mitochondrial DNA. Link to [WGS &delta;&sigma;](https://hubmapconsortium.github.io/ingest-validation-tools/wgs/ "directory schema"). _The consortium is no longer accepting data of this type_. |

#### Commentary:
Fellow **Documentation Team** members, this page is a "mock-up" of the _updated_ [HuBMAP Metadata by Dataset page](https://docs.hubmapconsortium.org/assays/metadata/)
- Suggested Options for the "attributes" icon: &#x2637;, &#x2338;, or &alpha;. My vote is **&alpha;**.
- Suggested Options for the "directory schema" icon: &#x0064;&#x0073;, &delta;&sigmaf;, &delta;&sigma;, or &Delta;&Sigma; - These are all variations of "DS" or "delta sigma," the Greek equivalent. If we go with **&alpha;** for attributes, I like **&delta;&sigma;**.
- We could also use a little icon (e.g. &#x2637; or &#x2338;), etc., or write out "directory schema".
- Anyone have any strong feelings about this choice?
