--- 
layout: page 
---
# Visium-HD

<details markdown="1"><summary>Version 2 (current)</summary>

## Version 2 (current)

| Attribute Name | Type | Description | Allowable Values | Required |
|---------------|------|-------------|------------------|----------|
| lab_id | Textfield | An internal field labs can use it to add whatever ID(s) they want or need for dataset validation and tracking. This could be a single ID (e.g., "Visium_9OLC_A4_S1") or a delimited list of IDs (e.g., “9OL; 9OLC.A2; Visium_9OLC_A4_S1”). This field will not be accessible to anyone outside of the consortium and no effort will be made to check if IDs provided by one data provider are also used by another. |  | False |
| preparation_protocol_doi | Link | DOI for the protocols.io page that describes the assay or sample procurement and preparation. For example for an imaging assay, the protocol might begin with staining of a section and finalize with the creation of an OME-TIFF file. In this case the protocol would include any image processing steps required to create the OME-TIFF file. Example: https://dx.doi.org/10.17504/protocols.io.eq2lyno9qvx9/v1. |  | True |
| dataset_type | Assigned Value | The specific type of dataset being produced. | ```Visium HD```, ```4i```, ```LC-MS```, ```Thick section Multiphoton MxIF```, ```Light Sheet```, ```ATACseq```, ```Resolve```, ```HiFi-Slide```, ```MPLEx```, ```10X Multiome```, ```MALDI```, ```Histology```, ```Cell DIVE```, ```FACS```, ```MS Lipidomics```, ```Visium (no probes)```, ```MUSIC```, ```RNAseq```, ```GeoMx (NGS)```, ```GeoMx (nCounter)```, ```RNAseq (with probes)```, ```Singular Genomics G4X```, ```Molecular Cartography```, ```CosMx Transcriptomics```, ```MERFISH```, ```Pixel-seqV2```, ```2D Imaging Mass Cytometry```, ```Confocal```, ```seqFISH```, ```DART-FISH```, ```MIBI```, ```Olink```, ```Enhanced Stimulated Raman Spectroscopy (SRS)```, ```DESI```, ```Xenium```, ```CyCIF```, ```SNARE-seq2```, ```nanoSPLITS```, ```Stereo-seq```, ```Visium (with probes)```, ```SIMS```, ```Auto-fluorescence```, ```CyTOF```, ```CosMx Proteomics```, ```DBiT-seq```, ```PhenoCycler```, ```CODEX```, ```Second Harmonic Generation (SHG)```, ```Seq-Scope``` | True |
| contributors_path | Textfield | The path to the file with the ORCID IDs for all contributors of this dataset (e.g., "./extras/contributors.tsv" or "./contributors.tsv"). This is an internal metadata field that is just used for ingest. |  | True |
| data_path | Textfield | The top level directory containing the raw and/or processed data. For a single dataset upload this might be "." where as for a data upload containing multiple datasets, this would be the directory name for the respective dataset. For instance, if the data is within a directory called "TEST001-RK" use syntax "./TEST001-RK" for this field. If there are multiple directory levels, use the format "./TEST001-RK/Run1/Pass2" in which "Pass2" is the subdirectory where the single dataset's data is stored. This is an internal metadata field that is just used for ingest. |  | True |
| mapped_area_value | Numeric | For Visium, this is the area of spots that was covered by tissue within the captured area, not the total possible captured area which is fixed. For GeoMx this would be the area of the AOI being captured. For HiFi this is the summed area of the ROIs in a single flowcell lane. For CosMx and Resolve, this is the area of the FOV (aka ROI) region being captured. For Xenium this is the total area of the FOV regions (aka ROI) being captured. For Stereo-Seq this is the number of beads. |  | True |
| mapped_area_unit | Assigned Value | The unit of measurement for the mapping area. For Visium and GeoMx this is typically um^2. | ```um^2```, ```mm^2``` | True |
| spot_size_value | Numeric | For assays where spots are used to define discrete capture areas, this is the area of a spot. |  | True |
| spot_size_unit | Assigned Value | The unit for spot size value. | ```um^2```, ```mm^2``` | True |
| number_of_spots | Numeric | Number of capture spots within the mapped area. For Visium this would be the number of spots covered by tissue, while it's the number of spots within ROIs for HiFi. |  | True |
| spot_spacing_value | Numeric | Approximate center-to-center distance between capture spots.  Synonyms: Inter-Spot distance, Spot resolution, Pit size |  | True |
| spot_spacing_unit | Assigned Value | Units corresponding to inter-spot distance | ```um``` | True |
| capture_area_id | Radio | Which capture area on the slide was used. For Visium this would be [A1, B1, C1, D1]. For HiFi this would be the lane on the flowcell. | ```A1```, ```B1```, ```C1```, ```D1```, ```Lane 1```, ```Lane 2```, ```Lane 3```, ```Lane 4```, ```Lane 5```, ```Lane 6```, ```Lane 7```, ```Lane 8``` | True |
| permeabilization_time_value | Numeric | Permeabilization time used for this tissue section. |  | False |
| permeabilization_time_unit | Assigned Value | The unit for the permeabilization time. | ```minute``` | False |
| metadata_schema_id | Textfield | The string that serves as the definitive identifier for the metadata schema version and is readily interpretable by computers for data validation and processing. Example: 22bc762a-5020-419d-b170-24253ed9e8d9 |  | True |
| parent_sample_id | Textfield | Unique HuBMAP or SenNet identifier of the sample (i.e., block, section or suspension) used to perform this assay. For example, for a RNAseq assay, the parent would be the suspension, whereas, for one of the imaging assays, the parent would be the tissue section. If an assay comes from multiple parent samples then this should be a comma separated list. Example: HBM386.ZGKG.235, HBM672.MKPK.442 or SNT232.UBHJ.322, SNT329.ALSK.102 |  | True |
| preparation_instrument_vendor | Assigned Value | The manufacturer of the instrument used to prepare (staining/processing) the sample for the assay. If an automatic slide staining method was indicated this field should list the manufacturer of the instrument. | ```Thermo Fisher Scientific```, ```SunChrom```, ```Leica Biosystems```, ```Roche Diagnostics```, ```In-House```, ```Not applicable```, ```Hamamatsu```, ```HTX Technologies```, ```10x Genomics``` | True |
| preparation_instrument_model | Assigned Value | Manufacturers of a staining system instrument may offer various versions (models) of that instrument with different features. Differences in features or sensitivities may be relevant to processing or interpretation of the data. | ```AutoStainer XL```, ```ST5020 Multistainer```, ```Visium CytAssist```, ```SunCollect Sprayer```, ```Chromium X```, ```Chromium iX```, ```EVOS M7000```, ```NanoZoomer S210```, ```NanoZoomer S60```, ```NanoZoomer S360```, ```Discovery Ultra```, ```Sublimator```, ```Not applicable```, ```TM-Sprayer```, ```M5 Sprayer```, ```M3+ Sprayer```, ```Chromium Controller```, ```Chromium Connect``` | True |
| non_global_files | Textfield | A semicolon separated list of non-shared files to be included in the dataset. The path assumes the files are located in the "TOP/non-global/" directory. For example, for the file is TOP/non-global/lab_processed/images/1-tissue-boundary.geojson the value of this field would be "./lab_processed/images/1-tissue-boundary.geojson". After ingest, these files will be copied to the appropriate locations within the respective dataset directory tree. This field is used for internal HuBMAP processing. Examples for GeoMx and PhenoCycler are provided in the File Locations documentation: https://docs.google.com/document/d/1n2McSs9geA9Eli4QWQaB3c9R3wo5d5U1Xd57DWQfN5Q/edit#heading=h.1u82i4axggee |  | False |

</details>